cfgmgr/tunnelmgr.cpp

```diff
@@ -260,7 +266,27 @@ bool TunnelMgr::doTunnelTask(const KeyOpFieldsValuesTuple & t)
              */
             if (m_tunnelReplay.find(tunnelName) == m_tunnelReplay.end())
             {
-                m_appIpInIpTunnelTable.set(tunnelName, kfvFieldsValues(t));
+                /* Create the tunnel */
+                std::vector<FieldValueTuple> fvs;
+                std::copy_if(kfvFieldsValues(t).cbegin(), kfvFieldsValues(t).cend(),
```

Comment:
What's the purpose of this `copy_if`?

Reply:
remove the `dst_ip`, as we are translating config db `TUNNEL` table into appl db `TUNNEL_DECAP_TABLE` and `TUNNEL_DECAP_TERM_TABLE`, the `dst_ip` is parsed to create decap terms in `TUNNEL_DECAP_TERM_TABLE`, `TUNNEL_DECAP_TABLE` should not have it anymore.